bin/NativeTests/JsRTApiTest.cpp

```diff
@@ -1428,15 +1429,15 @@ namespace JsRTApiTest
         REQUIRE(JsSetPromiseContinuationCallback(PromiseContinuationCallback, &callback) == JsNoError);
         REQUIRE(JsRunScript(
             _u("new Promise(") \
-            _u("  function(resolve, reject) {") \
-            _u("      resolve('basic:success');") \
-            _u("  }") \
+            _u("    function(resolve, reject) {") \
```

Comment:
There was one spacing issue with a brace before and that was just how I ended up fixing it. I can revert it back to two spaces.